app/assets/javascripts/pdfjs_viewer/viewer.js.erb

```diff
@@ -7141,17 +7141,19 @@ var PDFViewerApplication = {
 };
 window.PDFView = PDFViewerApplication; // obsolete name, using it as an alias
 
+var HOSTED_VIEWER_ORIGINS = <%= ENV.fetch("PDFJS_VIEWER_ORIGINS", "").split(",").inspect %>;
```

Comment:
This should have a comment to explain that it was changed. We need to reapply this change whenever we upgrade.